pasqal_cloud/__init__.py

```diff
@@ -435,6 +436,47 @@ def cancel_job(self, id: str) -> Job:
 
         return Job(**job_rsp, _client=self._client)
 
+    def cancel_jobs(
+        self,
+        batch_id: Union[UUID, str],
+        filters: Optional[CancelJobFilters] = None,
```

Comment:
Why not just default `filters` to `= CancelJobFilters()`? Then you can remove some of the logic below

Reply:
Because i think it is easier to understand it is optional like this.

Correct me if i am wrong but if I use this instead, I still need to make sure it is an instance of a Filter class passed right ?